gum/backend-x86/gumstalker-x86.c

```diff
@@ -31,7 +31,7 @@
 
 #define GUM_CODE_ALIGNMENT                     8
 #define GUM_DATA_ALIGNMENT                     8
-#define GUM_CODE_SLAB_SIZE_IN_PAGES        65536
+#define GUM_CODE_SLAB_SIZE_IN_PAGES         1024 // <ole> Erik Could you try lowering this: https://github.com/frida/frida-gum/blob/master/gum/backend-x86/gumstalker-x86.c#L34
```

Comment:
Would be good to do this in a separate one so I can squash the commits of this one easily. :)